hamlpy/parser/filters.py

```diff
@@ -31,67 +37,69 @@
 from future.utils import raise_from
 
 from .core import ParseException
+from .utils import html_escape
+
+
+# ----------------------------------------------------------------------------------
+# Core filters
+# ----------------------------------------------------------------------------------
+
+def plain(text, options):
+    return text
```

Comment:
Currently each filter handles indentation separately but now, like the Ruby implementation, indentation is handled outside of the filter.